app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -4,6 +4,7 @@ import notesApi from "apis/notes";
 
 export default function DeleteAlert({ refetch, onClose, selectedNoteIds }) {
```

Comment:
Please replace this file and use neetoUI [Alert Component](https://neeto-ui.netlify.app/#/alert) instead.